### PR TITLE
Feature/about

### PR DESCRIPTION
**Pull request by Mrichk0**

### CODE REVIEW

File: src/partials/header.html [hunk 1]
```diff
@@ -1,7 +1,98 @@
...
+          <svg
+            class="btn-list__icon"
+            width="7"
+            height="10"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="m1.42 1 4.282 4L1.42 9"
+              stroke="#D41443"
```
Comment (VovaRubakha) [inline src/partials/header.html]: var

File: src/partials/header.html [hunk 2]
```diff
@@ -1,7 +1,98 @@
...
+          <li><a class="header__list" href="">Home</a></li>
+          <li><a class="header__list" href="">About us</a></li>
+          <li><a class="header__list" href="#How">How it&#39;s made?</a></li>
+          <li><a class="header__list" href="">Our products</a></li>
+          <li><a class="header__list" href="">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+    <button type="button" class="header__button is-open" aria-expanded="false" data-header__button>
+      <svg width="20" height="20" aria-label="mobil menu ">
```
Comment (VovaRubakha) [inline src/partials/header.html]: change to  aria-label="mobile menu"

File: src/partials/header.html [hunk 3]
```diff
@@ -1,7 +1,98 @@
...
-  <h1 class="header-title">Hello world</h1>
-  <p class="lorem">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Placeat, doloribus.</p>
-  <svg class="" width="50px" height="50px">
-    <use href="./images/icons.svg#icon-ice-cream"></use>
-  </svg>
+<header class="header">
+  <div class="container header__container">
+    <div class="header__backdrop is-hidden" data-modal>
+      <nav class="header__menu" data-menu>
+        <button class="header__menu--button">
```
Comment (VovaRubakha) [inline src/partials/header.html]: type="button"

File: src/partials/header.html [hunk 4]
```diff
@@ -1,7 +1,98 @@
...
+        </ul>
+      </nav>
+    </div>
+    <button type="button" class="header__button is-open" aria-expanded="false" data-header__button>
+      <svg width="20" height="20" aria-label="mobil menu ">
+        <use class="header__icon--cross" href="./images/icons.svg#icon-cross"></use>
+        <use class="header__icon--menu" href="./images/icons.svg#icon-burger"></use>
+      </svg>
+    </button>
+    <button class="header__nav--button">
```
Comment (VovaRubakha) [inline src/partials/header.html]: type="button"

File: src/partials/header.html [hunk 5]
```diff
@@ -1,7 +1,98 @@
...
+      <svg
+        class="btn-list__icon"
+        width="7"
+        height="10"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="m1.42 1 4.282 4L1.42 9"
+          stroke="#D41443"
```
Comment (VovaRubakha) [inline src/partials/header.html]: var

File: src/partials/header.html [hunk 6]
```diff
@@ -1,7 +1,98 @@
...
+    >
+    <!-- <a href="./hero.html">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+          width="200"
+        />
+      </a> -->
+    <h2 class="header__taitle">
```
Comment (VovaRubakha) [inline src/partials/header.html]: "header__title"

File: src/partials/header.html [hunk 7]
```diff
@@ -1,7 +1,98 @@
...
+      ice cream made with
+      <span class="header__taitle--emphasis">passion</span>
```
Comment (VovaRubakha) [inline src/partials/header.html]: "header__title"

File: src/sass/layouts/_header.scss [hunk 8]
```diff
@@ -1,6 +1,298 @@
...
+  // @media screen and (min-width: 1280px) {
+  //   margin-bottom: 94px;
+  //   //стерти
+  //   margin-top: -50px;
+  // }
+}
+.header__logo--icon {
+  margin-right: 17px;
+}
+.header__taitle {
```
Comment (VovaRubakha) [inline src/sass/layouts/_header.scss]: "header__title"

File: src/sass/layouts/_header.scss [hunk 9]
```diff
@@ -1,6 +1,298 @@
...
+  @media screen and (min-width: 1280px) {
+    width: 277px;
+    font-size: 38px;
+    line-height: 1.13;
+    padding-top: 85px;
+    // padding-left: 114px;
+    // padding-top: 80px;
+  }
+}
+.header__taitle--emphasis {
```
Comment (VovaRubakha) [inline src/sass/layouts/_header.scss]: "header__title"

File: src/sass/components/_mobile-menu.scss [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
+  @media screen and (max-width: 1279px) {
+    position: absolute;
+    display: none;
+    transform: translateX(-100%);
+    transition: transform 8500ms;
+    top: 0;
+    right: 0;
+    width: 255px;
+    height: 100%;
+    background-color: #e17992;
```
Comment (VovaRubakha) [inline src/sass/components/_mobile-menu.scss]: var

File: src/sass/components/_mobile-menu.scss [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+    display: none;
+  }
+}
+
+.header__nav {
+  @media screen and (min-width: 1280px) {
+    display: flex;
+    justify-content: center;
+    padding-top: 10px;
+    @media screen and (max-width: 1279px) {
```
Comment (VovaRubakha) [inline src/sass/components/_mobile-menu.scss]: що це і куда?

File: src/sass/components/_mobile-menu.scss [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+}
+.header__list {
+  display: block;
+  text-decoration: none;
+  font-family: DM Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 1.28;
+  color: #ffffff;
```
Comment (VovaRubakha) [inline src/sass/components/_mobile-menu.scss]: var

File: src/sass/components/_mobile-menu.scss [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+  }
+  @media screen and (min-width: 1280px) {
+    font-size: 16px;
+    line-height: 21px;
+    margin-right: 30px;
+  }
+  // &:not(:last-child) {
+  //   margin-right: 30px;
+  // }
+  &:hover {
```
Comment (VovaRubakha) [inline src/sass/components/_mobile-menu.scss:68]: add focus

File: src/sass/components/_mobile-menu.scss [hunk 14]
```diff
@@ -0,0 +1,170 @@
...
+    color: #c7133e;
```
Comment (VovaRubakha) [inline src/sass/components/_mobile-menu.scss]: var

**Review by VovaRubakha — CHANGES_REQUESTED** [review]

File: src/sass/layouts/_header.scss [hunk 15]
```diff
@@ -1,6 +1,298 @@
...
+  font-size: 22px;
+  line-height: 25px;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  color: #d41443;
+  text-decoration: none;
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 137px;
```
Comment (VovaRubakha) [inline src/sass/layouts/_header.scss]: no need, delete

File: src/sass/layouts/_header.scss [hunk 16]
```diff
@@ -1,6 +1,298 @@
...
+  font-size: 8px;
+  line-height: 1.25;
+  color: #ffffff;
+  @media screen and (min-width: 1280px) {
+    width: 104px;
+    height: 40px;
+    font-size: 14px;
+    line-height: 1.28;
+  }
+  &:hover {
```
Comment (VovaRubakha) [inline src/sass/layouts/_header.scss:273]: add focus

File: src/sass/layouts/_header.scss [hunk 17]
```diff
@@ -1,6 +1,298 @@
...
+  font-size: 8px;
+  line-height: 1.25;
+  color: #d41443;
+  @media screen and (min-width: 1280px) {
+    width: 138px;
+    height: 40px;
+    font-size: 14px;
+    line-height: 1.28;
+  }
+  &:hover {
```
Comment (VovaRubakha) [inline src/sass/layouts/_header.scss:296]: add focus

**Review by VovaRubakha — CHANGES_REQUESTED** [review]

**Comment by Mrichk0** [comment]
@VovaRubakha  done

**Review by VovaRubakha — APPROVED** [review]

**Pull request merged**